.github/pull_request_template.md

```diff
@@ -5,23 +5,23 @@
 ## PR Checklist
 
 - [ ] Have you verified the basic functionality for this change?
-
-- [ ] Added unit tests?
-
-- [ ] Added e2e tests
-
+- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
+- [ ] Does this PR include changes which are untested or may affect areas with [known gaps in automated testing](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2630719564/Automation+Gaps)? (If yes, provide test plan below)
 - [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
+- [ ] Did you verify the change by running downstream monolith artifacts? (Provide link to build below)
```

Comment:
Do we have a link with instructions on how to run a d16t? Not sure if every contributor knows

Reply:
Just got this question from an eng who didn't know how to trigger the d16t build.
Can we paste instructions from here https://github.com/okta/okta-core/pull/68408 into a wiki and link it?

Reply:
We have instructions here: https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t)
Adding to PR template